packages/scaffolding-cli/src/domain/config/worker_maps/shared.ts

```diff
@@ -95,7 +95,7 @@ ${ranAdvanced ? advanced : basic}
 
 🤓 To get started: open ${dir}/README.md
 
-📖 For guides and supporting information see: <url>
+📖 For guides and supporting information see: https://amido.github.io/stacks/
```

Comment:
👍 also there is a lib that makes links clickable in the terminal of certain types we can wrap it in that

Reply:
Good shout, mine does this automatically? Think most terminals do if prefixed with http/https?